go-controller/pkg/ovn/common.go

```diff
@@ -25,6 +25,30 @@ func hashedPortGroup(s string) string {
 	return hashForOVN(s)
 }
 
+func findAddressSet(name string, hashName string) (string, error) {
+	uuid, stderr, err := util.RunOVNNbctl("--data=bare",
+		"--no-heading", "--columns=_uuid", "find", "address_set",
+		"name="+hashName)
+	if err != nil {
+		return "", fmt.Errorf("find failed to get address set %q, stderr: %q (%v)",
+			name, stderr, err)
+	}
+
+	return uuid, nil
+}
+
+func findPortGroup(name string, hashName string) (string, error) {
+	portGroup, stderr, err := util.RunOVNNbctl("--data=bare",
```

Comment:
is it possible to use the port group name as opposed to the UUID like we do for address sets?